src/eosjs2-api.ts

```diff
@@ -93,26 +93,41 @@ export class Api {
         }));
     }
 
```

Comment:
pushTransaction should be kept and console.warn future deprecation, internally calling sign and push transaction

Reply:
This would also avoid major revision change and instead be a minor

Reply:
I had originally kept the functionality of pushTransaction the same but because we wanted to eventually use the name "pushTransaction" for its' true purpose (to simply push/broadcast, not sign and push) Todd said it was preferred to make this PR a breaking change. One thing I could do to make this a minor change and maintain the split functionality is to rename pushTransaction to broadcastTransaction then have pushTransaction do the console.warn and internally call the newly created signAndPushTransaction function... @chris-allnutt @tbfleming thoughts?